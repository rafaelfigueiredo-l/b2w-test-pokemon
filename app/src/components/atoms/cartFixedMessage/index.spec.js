import CartFixedMessage from "./";

describe("<CartFixedMessage />", () => {
  const component = <CartFixedMessage>Teste</CartFixedMessage>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly CartFixedMessage component", () => {
    expect(wrapper.find(".cartFixedMessage")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly CartFixedMessage component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".cartFixedMessage").children()).toHaveLength(1);
  });
});
