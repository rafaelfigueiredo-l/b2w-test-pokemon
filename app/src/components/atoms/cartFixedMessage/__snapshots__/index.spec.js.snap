// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartFixedMessage /> should mount correctly CartFixedMessage component 1`] = `
<styled.p
  className="cartFixedMessage"
>
  <p
    className="sc-AxjAm ehpYJS cartFixedMessage"
  >
    Teste
  </p>
</styled.p>
`;
